plugin/src/main/java/org/opensearch/ml/action/memorycontainer/memory/MemoryProcessingService.java

```diff
@@ -307,6 +309,10 @@ private List<String> parseFactsFromLLMResponse(MemoryStrategy strategy, MLOutput
             return facts;
         }
 
+        List<Map<String, Object>> processorConfigs = new ArrayList<>();
+        processorConfigs.add(Map.of("type", MLProcessorType.EXTRACT_JSON.getValue(), "extract_type", "object"));
+        ProcessorChain processorChain = new ProcessorChain(processorConfigs);
```

Comment:
If this is the default/required processorChain, why not initialize it as the class member and use it directly. I think this processChain doesn't contain any memory data so there's no need to create a new ProcessorChain for each memory request.